### PR TITLE
DSA-Day10

### PR DESCRIPTION
**Pull request by vaibhav721**

### CODE REVIEW

File: data-structures/tree/binary-tree/Traversals/Inorder/recursive_inorder_travesal.cpp [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#include <iostream>
```
Comment (InclinedScorpio) [inline data-structures/tree/binary-tree/Traversals/Inorder/recursive_inorder_travesal.cpp:1]: please add question

File: data-structures/tree/binary-tree/Inversion_of_tree.cpp [hunk 2]
```diff
@@ -0,0 +1,82 @@
+#include <iostream>
```
Comment (InclinedScorpio) [inline data-structures/tree/binary-tree/Inversion_of_tree.cpp:1]: please add question

Comment (vaibhav721) [reply data-structures/tree/binary-tree/Inversion_of_tree.cpp:2]: Added

File: data-structures/tree/binary-tree/Traversals/Inorder/recursive_inorder_travesal.cpp (hunk 1 above)
Comment (vaibhav721) [reply data-structures/tree/binary-tree/Traversals/Inorder/recursive_inorder_travesal.cpp:2]: added

**Review by InclinedScorpio — APPROVED** [review]
Thanks !! 🎉

**Pull request merged**